source/MapPanel.cpp

```diff
@@ -1105,14 +1128,16 @@ void MapPanel::DrawWormholes()
 void MapPanel::DrawLinks()
 {
 	double zoom = Zoom();
+	if(Zoom() <= 0.5 && isStarry == false)
+		return;
 	for(const Link &link : links)
 	{
 		Point from = zoom * (link.start + center);
 		Point to = zoom * (link.end + center);
-		Point unit = (from - to).Unit() * LINK_OFFSET;
+		float linkOffset = (isStarry == true) ? LINK_OFFSET : 0;
+		Point unit = (from - to).Unit() * linkOffset;
```

Comment:
Here you can fetch the linkoffset directly instead of storing it into a variable; as before

Also you can write `isStary` instead of isStaru == true